iocore/hostdb/P_HostDBProcessor.h

```diff
@@ -208,29 +204,17 @@ struct RefCountedHostsFileMap : public RefCountObj {
 //
 // HostDBCache (Private)
 //
-struct HostDBCache : public MultiCache<HostDBInfo> {
-  int rebuild_callout(HostDBInfo *e, RebuildMC &r);
+struct HostDBCache {
   int start(int flags = 0);
-  MultiCacheBase *
-  dup()
-  {
-    return new HostDBCache;
-  }
-
-  // This accounts for an average of 2 HostDBInfo per DNS cache (for round-robin etc.)
-  // In addition, we can do a padding for additional SRV records storage.
-  // In addition, we add 120 for hostname storage (since we now always do that)
-  virtual size_t
-  estimated_heap_bytes_per_entry() const
-  {
-    return sizeof(HostDBInfo) * 2 + 512 * hostdb_srv_enabled + 120;
-  }
-
   // Map to contain all of the host file overrides, initialize it to empty
   Ptr<RefCountedHostsFileMap> hosts_file_ptr;
+  // TODO: cleanup
+  RefCountCache<HostDBInfo> *refcountcache;
```

Comment:
If ``refcountcache`` was ``Ptr<RefCountCache>>`` it will get cleaned up automatically.